source/ota.c

```diff
@@ -2982,7 +2982,7 @@ OtaErr_t OTA_SetImageState( OtaImageState_t state )
             /*
              * Set the image state as rejected.
              */
-            err = setImageStateWithReason( state, 0 );
```

Comment:
Can 0 be replaced with the enum? I think this should be the OtaErrNone OtaErr value. Just in case this value changes later. Also applies to line 2994. This can be in a separate PR if you want since the point of this one is just for MISRA.